debug/test/browser/debug.test.js

```diff
@@ -84,7 +104,7 @@ describe('debug', () => {
 				<p>{state}</p>
 			);
 		};
-		const fn = () => render(<App />, scratch);
+		const fn = () => act(() => render(<App />, scratch));
```

Comment:
What is the effect of adding the `act` call here?

Reply:
Hooks inside the render method of App will be flushed